ci/travis-build.sh

```diff
@@ -11,6 +11,7 @@ case "$TRAVIS_OS_NAME" in
         case "$CI_TARGET_ARCH" in
             "x86")
                 EXTRA_CFLAGS="${EXTRA_CFLAGS} -m32"
+                export LDFLAGS="-m32"
```

Comment:
Shouldn't this patch be folded into the patch that removes CFLAGS from the linker commands?

Reply:
Done.